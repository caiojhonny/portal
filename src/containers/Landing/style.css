.landing--wrapper {
  flex: 1;
  justify-content: space-between;
  align-items: center;
}

.landing--block {
  display: flex;
  flex-direction: column;
  align-self: center;
  align-items: center;
  margin-top: 50%;
}

.landing--logo {
  width: 100px;
  height: 75px;
}

.landing--title {
  display: block;
}

.landing--subtitle {
  display: block;
  font-size: 0.7rem;
}

.graphql--section {
  text-align: center;
  margin-top: 50%;
}

