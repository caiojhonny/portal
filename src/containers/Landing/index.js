import React, { PureComponent } from 'react'

import './style.css'

class Landing extends PureComponent {
  render () {
    return (
      <div className='landing--wrapper'>
        <div className='landing--block'>
          <img className='landing--logo' src={require('../../assets/images/logo_beta.png')} />
          <span className='landing--title'>Work with Coffee</span>
          <span className='landing--subtitle'>A top curated list of work places for you.</span>
        </div>
      </div>
    )
  }
}

export default Landing
