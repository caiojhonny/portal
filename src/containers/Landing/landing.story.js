import { storiesOf } from '@storybook/react'
import React from 'react'
import Landing from './index'

const stories = storiesOf('Landing', module)

stories.add('Landing temp page', () => (
  <Landing />
))
