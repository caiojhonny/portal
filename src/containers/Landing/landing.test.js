
import React from 'react'
import renderer from 'react-test-renderer'
import Landing from './index'

it('Should Landing match to snapshot', () => {
  const tree = renderer.create(<Landing />).toJSON()
  expect(tree).toMatchSnapshot()
})
