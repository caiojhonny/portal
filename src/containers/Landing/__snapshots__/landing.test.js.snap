// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Landing match to snapshot 1`] = `
<div
  className="landing--wrapper"
>
  <div
    className="landing--block"
  >
    <img
      className="landing--logo"
      src={Object {}}
    />
    <span
      className="landing--title"
    >
      Work with Coffee
    </span>
    <span
      className="landing--subtitle"
    >
      A top curated list of work places for you.
    </span>
  </div>
</div>
`;
